### ISSUE TITLE
Load management should be 'off' by default and the new profiles should be used

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etmoses#194]
This has to happen before Monday 7-6-2015 :worried:

* The first thing is to have load management 'off' by default. It would be nice if it could be toggled but that is a nice-to-have.

* Secondly, the new profiles should be used if possible. For base-load, they consist of a flexible and an inflexible part. Both parts should be **added** when load management is 'off'. The rest likely works out of the box (right @antw ?)

**Comment by grdw** [comment]
> The first thing is to have load management 'off' by default. It would be nice if it could be toggled but that is a nice-to-have

How can I tell if it's turned on right now?

**Comment by ChaelKruip** [comment]
> How can I tell if it's turned on right now?

I don't know. @antw told me it was on :smile:

**Comment by antw** [comment]
[On this line](https://github.com/quintel/etmoses/blob/c054d55c75f9e7af38bbe013b821d1a5cc716e5e/app/controllers/testing_grounds_controller.rb#L67), "flexibility" is set to true. To turn it off, set it to false.

> The rest likely works out of the box (right @antw ?)

Yep! The inflexible part will need to be a new `Technology` added to the database; set its `behavior` attribute to "optional" to get the expected result.
